src/components/GenerateKeypair.tsx

```diff
@@ -0,0 +1,37 @@
+import { Icon, Input } from "@stellar/design-system";
+import { useStore } from "@/store/useStore";
+
+export const GenerateKeypair = ({ secretKey }: { secretKey: string }) => {
```

Comment:
Should we make this component not connected to the store? I mean, pass in public and secret keys for it to display, and that's it.